apps/laboratory/src/utils/DataUtil.ts

```diff
@@ -109,3 +116,60 @@ export const solanaSdkOptions = [
     description: 'Configuration using ethers without socials'
   }
 ]
+
+export const multichainSdkOptions: SdkOption[] = [
+  {
+    title: 'Wagmi + Solana',
+    link: '/library/multichain-wagmi-solana',
+    description: 'Configuration with Wagmi and Solana adapters enabled for AppKit'
+  },
+  {
+    title: 'Ethers + Solana',
+    link: '/library/multichain-ethers-solana',
+    description: 'Configuration with Ethers and Solana adapters enabled for AppKit'
+  },
+  {
+    title: 'Ethers5 + Solana',
+    link: '/library/multichain-ethers5-solana',
+    description: 'Configuration with Ethers 5 and Solana adapters enabled for AppKit'
+  },
+  {
+    title: 'Basic',
+    link: '/library/multichain-basic',
+    description: 'Configuration with no adapters enabled for AppKit'
+  }
+]
+
+export const testingSdkOptions: SdkOption[] = [
+  {
+    title: 'Demo',
+    link: '',
+    description: 'All features enabled and randomly using ethers or wagmi',
+    randomLinks: ['/library/wagmi-all', '/library/ethers-all']
+  },
+  {
+    title: 'Demo w/ Sample Wallets',
+    link: '',
+    description:
+      'All features enabled, with sample wallet links, and randomly using ethers or wagmi',
+    randomLinks: ['/library/wagmi-all-internal', '/library/ethers-all-internal']
+  }
+]
+
+export const featuredSdkOptions: SdkOption[] = [
+  {
+    title: 'Default',
+    link: '/library/wagmi',
+    description: 'Basic configuration using wagmi as a driving library'
+  },
```

Comment:
should we add ethers as well?

Reply:
I don't think so, this is mostly for people showcasing our offerings. Using wagmi or ethers is irrelevant.